### ISSUE TITLE
Make the "Now Playing" slot work with podcasts

### ISSUE THREAD

**Issue by InDaZiploc** [BluDood/GlanceThing#29]
I use spotify most for podcasts, but they don't show up on the GlanceThing; it would be much appreciated if podcasts showed up.

**Comment by BluDood** [comment]
Good idea, haven't really used podcasts in Spotify so never tested that it worked. Will be implemented!

**Comment by BluDood** [comment]
After some further research I can see that the Spotify WebSocket API (which is providing live playback updates) does send an event when a podcast is played, but it doesn't actually send any information about the actual podcast. A workaround could just be to fetch the current playing podcast when the event is received, which is still not hammering the Spotify API because it only fetches info when there's an actual update

**Comment by BluDood** [comment]
Ended up going with this solution, and seems to work fine. Playing a podcast also removes the shuffle/repeat buttons from the fullscreen player since those aren't avaiable in the official Spotify client. 

![Image](https://github.com/user-attachments/assets/b2df0d83-1029-4dd3-9422-95f1a273d4f7)

![Image](https://github.com/user-attachments/assets/7677206c-15a7-4b9f-a367-80c663d46656)

Fixed in cf65e078952d8c90eb25edaf032a5624c48272af
